numba/tests/test_np_randomgen.py

```diff
@@ -67,7 +77,7 @@ def check_numpy_parity(self, distribution_func,
                                               test_size, test_dtype)
 
         np.testing.assert_array_max_ulp(numpy_res, numba_res,
-                                        maxulp=5, dtype=test_dtype)
+                                        maxulp=ulp_prec, dtype=test_dtype)
```

Comment:
Could this be a kwarg on the `check_numpy_parity` method, else all distributions are tested to the relaxed level when they need not be?